app/javascript/packages/verify-flow/steps/password-confirm/forgot-password.tsx

```diff
@@ -0,0 +1,45 @@
+import { useContext } from 'react';
+import { PageHeading, Button } from '@18f/identity-components';
+import { t } from '@18f/identity-i18n';
+import { getAssetPath } from '@18f/identity-assets';
+import { VerifyFlowContext } from '@18f/identity-verify-flow';
+
+interface ForgotPasswordProps {
+  goBack: () => void;
+}
+
+export function ForgotPassword({ goBack }: ForgotPasswordProps) {
+  const { resetPasswordUrl } = useContext(VerifyFlowContext);
+
+  function goToResetPassword() {
+    window.location.href = resetPasswordUrl!;
+  }
+
+  return (
+    <>
+      <img
+        src={getAssetPath('status/info-question.svg')}
+        width="50"
+        alt={t('components.status_page.icons.question')}
+        height="50"
```

Comment:
This image is 54x54 in the production view.

```suggestion
        width="54"
        height="54"
        alt={t('components.status_page.icons.question')}
```

Reference:

https://github.com/18F/identity-idp/blob/37e8d8b9d31ade5258d35b736970f32338a2d74c/app/components/status_page_component.html.erb#L1